source/dapp-development/index.rst

```diff
@@ -47,7 +47,7 @@ Aergo also offers a development toolchain that includes package management, `Aer
 Infrastructure
 --------------
 
-Aergo is currently available as a `public testnet <../using-testnet/index.html>`_ and separate private installations.
+Aergo is currently available as a `public testnet <../using-network/connecting.rst#testnet>`_ and separate private installations.
```

Comment:
The correct is `https://docs.aergo.io/en/latest/using-network/connecting.html#testnet`
html instead of rst